src/main/groovy/liquibase/harness/base/BaseCompatibilityTestHelper.groovy

```diff
@@ -36,6 +40,32 @@ class BaseLevelTestHelper {
         return inputList
     }
 
+    static ResultSet executeQuery(String pathToSql, TestInput testInput) throws SQLException {
+        Connection newConnection
+        ResultSet resultSet
```

Comment:
Reuse same connection from testInput.database.connection. Reopen it if it's closed and assign to same var

Reply:
++